fastlane/lib/fastlane/plugins/templates/action.rb.erb

```diff
@@ -0,0 +1,35 @@
+module Fastlane
+  module Actions
+    class <%= plugin_name.capitalize %>Action < Action
+      def self.run(params)
+        UI.message("The <%= plugin_name %> plugin is working!")
+      end
+
+      def self.description
+        %q{<%= summary %>}
```

Comment:
Yes, the last point is fair. I guess `%q` just requires less work for the likely case. I believe we copied the strategy from Cocoapods, for what that's worth.

I guess my point about trying to do escaping is that it is always more tricky than you initially think. For example given the user input that results in:

```
My description is either "good"\"bad"
```

Just escaping `"` leads to:

```
My description is either \"good\"\\"bad\"
```

which I believe will be broken (the double backslash escapes the backslash, not the quotes) if you place inside of a 
```
"<%= template %>"
```